packages/core/src/shared/utilities/pathFind.ts

```diff
@@ -30,10 +30,12 @@ export async function tryRun(
     p: string,
     args: string[],
     logging: 'yes' | 'no' | 'noresult' = 'yes',
-    expected?: string
+    expected?: string,
+    onStdout?: (proc: ChildProcess, text: string) => void
```

Comment:
probably should just accept a `opt: ChildProcessOptions`

Reply:
Is there a way to do this without modifying the `ChildProcessOptions` interface? I need the `ChildProcess` object itself in-scope for the `onStout` definition, and couldn't figure out how to do this with the current setup in `src/shared/utilities/childProcess.ts`.

Reply:
oh i see. ok, can revisit this later if needed.

Reply:
a low-churn way to do this would be to add a `send()` method to `RunParameterContext`, similar to the existing `stop()` method: https://github.com/aws/aws-toolkit-vscode/blob/feb62bb953e62534df5e32731b8c989e9dfb9f4c/packages/core/src/shared/utilities/processUtils.ts#L12-L16

then I think just add `send: this.send.bind(this)` here  : https://github.com/aws/aws-toolkit-vscode/blob/feb62bb953e62534df5e32731b8c989e9dfb9f4c/packages/core/src/shared/utilities/processUtils.ts#L160-L163